src/app/account/signin/sign-in-form.tsx

```diff
@@ -94,32 +95,28 @@ export const SignInForm: FC<{
                         key={form.key('password')}
                         {...form.getInputProps('password')}
                         mt={10}
-                        mb="xs"
                         placeholder="*********"
                         aria-label="Password"
                     />
-                    {clerkError && (
-                        <Paper bg="#FFEFEF" shadow="none" p={'lg'} mt="sm" mb="sm" radius="sm">
-                            <Group justify="space-between" gap="xl">
-                                <Text ta="left" c="red" fw="bold">
-                                    {clerkError.title}
-                                </Text>
-                                <CloseButton
-                                    c="red"
-                                    aria-label="Close password reset form"
-                                    onClick={() => setClerkError(null)}
-                                />
-                            </Group>
-                            <Text mt="md">{clerkError.message}</Text>
-                        </Paper>
-                    )}
                     <Link
+                        c="blue.7"
+                        fw={600}
+                        w="fit-content"
+                        size="xs"
                         href={`/account/reset-password${searchParams.get('redirect_url') ? `?redirect_url=${searchParams.get('redirect_url')}` : ''}`}
                     >
                         Forgot password?
                     </Link>
+                    {clerkError && <SignInError clerkError={clerkError} setClerkError={setClerkError} />}
```

Comment:
instead of wrapping in `{clerkError &&` you can have `SignInError` return null if clerkError is falsy